src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -10,7 +11,11 @@
 import net.sf.jabref.model.entry.ParsedFileField;
 
 import java.io.File;
-import java.util.*;
```

Comment:
I think the sorting of the imports differs from https://github.com/JabRef/jabref/issues/1091